client/src/components/PostDetails.jsx

```
@@ -39,6 +39,12 @@ export default function PostDetails()
        
         subscribeToUser(newSubscription).then(() => {window.alert(`You are now subscribed to ${post.author?.firstName}`)})
     }
+
+    const handleUnsubscribeClick = () => {
+        const followerId = user.id
+        const subscriberId = post.author?.id
```

I'd remove the ? here. In the case that author is undefined, we want that to throw an error instead of setting `subscriberId` to undefined. This is because the case where you'd get an error for `author.id` not being defined would be easier to debug than the case of the fetch failing because undefined was passed. 

This is called the fail fast principle. If you're curious to learn more, [here's](https://medium.com/@christian.ppl/introduction-to-the-fail-fast-principle-in-software-development-865ccab28979) a great article about it.

---

changes implemented